src/org/zaproxy/zap/extension/portscan/resources/Messages.properties

```diff
@@ -2,7 +2,7 @@
 
 ports.copy.popup                = Copy
 ports.desc                      = Simple but effective port scanner
-ports.options.label.maxPort     = Maximum port to scan:
+ports.options.label.maxPort     = Maximum port(s) to scan:
```

Comment:
Isnt this the highest port number that will be scanned, in which case shouldnt it be singular rather than plural?